source/help/getting-started/signing-in.rst

```diff
@@ -0,0 +1,125 @@
+Signing in
+==========
+
+--------------
+
+You can sign in to your team from the root page of your Mattermost site
+``https://domain.com``. After signing in, you will be able to view a
+list of your teams and select which one to open.
+
+Sign In Methods
+---------------
+
+There are several options for signing in to your team depending on how
+your System Admin has configured your server.
+
+Email Address or Username Sign In
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When enabled by your System Admin, you can sign in with the username or
+email address used to create your account.
+
+If you have forgotten your password, you can reset it by clicking **I
+forgot my password** on the sign in screen, or contact your System Admin
+for help resetting your password.
+
+GitLab Single-Sign-On (SSO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When enabled by your System Admin, you can sign in using your GitLab
+account using a one-click sign in option.
+
+Google Single-Sign-On
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*Available in Enterprise Edition E20*
+
+When enabled by your System Admin, you can sign in using your Google
+account using a one-click sign in option.
+
+Office 365 Single-Sign-On (Beta)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*Available in Enterprise Edition E20*
+
+When enabled by your System Admin, you can sign in using your Office 365
+account using a one-click sign in option.
+
+AD/LDAP Sign In
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*Available in Enterprise Edition E10 & E20*
+
+When enabled by your System Admin, you can sign in with your AD/LDAP
+credentials. This lets you use the same username and password for
+Mattermost that you use for various other company services.
+
+SAML Single-Sign-On (SSO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*Available in Enterprise Edition E20*
+
+When enabled by your System Admin, you can sign in with your SAML
+credentials. This lets you use the same username and password for
+Mattermost that you use for various other company services. Mattermost
+officially supports `Okta`_ and `Microsoft ADFS`_ as an identity
+provider (IDP) for SAML, but you may use other SAML IDPs as well. Please
+see `documentation`_ to learn more about configuring SAML for
+Mattermost.
+
+Switching Teams
+---------------
+
+You can switch between teams you have joined using the **Main Menu**,
+accessed by clicking the three dots in the top header of any team site
+on the server.
+
+Logging Out
+-----------
+
+You can log out from the **Main Menu**, which is accessed by clicking
+the three dots in the top header on the left side of the screen.
+Clicking “Logout” logs you out of all teams that are signed in and open
```

Comment:
Should we also update this to just say "logs you out of all teams on the server"?